### DISCUSSION TITLE
Audit log

### DISCUSSION THREAD

**Discussion by Phoenixian** [inventree/InvenTree#5932] [Q&A] [answered]
Hello,
Does Inventree have an audit log to trace all any actions of all users ?
Thanks.

**Comment by matmair** [comment, marked as answer]
No

**Comment by SchrodingersGat** [comment]
@Phoenixian it has been discussed in the past, but insufficient developer time / resource to implement a sufficient solution

**Reply by matmair** [reply]
@Phoenixian a good starting point would be to submit a detailed feature request so interested developers can get a feeling for the scope and use case.

**Comment by Phoenixian** [comment]
Thank you for your reply. It seems to me that this functionality is essential in professional and multi-user use, in particular to know who took an object out of stock, when and why. I'm making a feature request. Thanks again, great software!

**Reply by matmair** [reply]
We already track stock movement very detailed. You asked for any action by all users - which we do not provide.
